pinot-controller/src/test/java/org/apache/pinot/controller/recommender/TestConfigEngine.java

```diff
@@ -229,6 +230,10 @@ void testPinotTablePartitionRule()
       throws InvalidInputException, IOException {
     loadInput("recommenderInput/PinotTablePartitionRuleInput.json");
 
+    // segment size recommendations get populated by SegmentSize Rule; hard-coding the values here
+    _input._overWrittenConfigs.setSegmentSizeRecommendations(
+        new SegmentSizeRecommendations(/*numRows=*/1_000_000, /*numSegments=*/4, /*segmentSize=*/1_000_000));
+
```

Comment:
This also reinforces the point that it is better to always run this rule. Wdyt ?

Reply:
Refer to prev. comment.